README.md

```diff
@@ -32,11 +42,10 @@ https://registry.hub.docker.com/u/antonlindstrom/mesos-exporter/
 If you want to use it with your own configuration, you can mount it as a
 volume:
 
-    docker run -d -v /root/config.json:/config.json -p 4000:4000 antonlindstrom/mesos-exporter
+    docker run -d -p 4000:4000 antonlindstrom/mesos-exporter
```

Comment:
I guess this example and the below should be changed to be `prom/mesos-exporter` now?